front/src/modules/ui/table/options/components/TableOptionsDropdownButton.tsx

```diff
@@ -33,17 +53,37 @@ enum Option {
 
 export const TableOptionsDropdownButton = ({
   onColumnsChange,
+  onViewsChange,
   HotkeyScope,
 }: TableOptionsDropdownButtonProps) => {
   const theme = useTheme();
+
   const [isUnfolded, setIsUnfolded] = useState(false);
   const [selectedOption, setSelectedOption] = useState<Option | undefined>(
     undefined,
   );
 
+  const viewEditInputRef = useRef<HTMLInputElement>(null);
```

Comment:
I am not sure we need this one, we could pass the right view name as a parameter of handleViewNameSubmit instead?